firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -0,0 +1,137 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class InactiveTabsTest: BaseTestCase {
+    override func setUp() {
+        launchArguments = [
+            LaunchArguments.PerformanceTest,
+            LaunchArguments.SkipIntro,
+            LaunchArguments.SkipWhatsNew,
+            LaunchArguments.SkipETPCoverSheet,
+            LaunchArguments.SkipContextualHints,
+            LaunchArguments.DisableAnimations
+        ]
+
+        // Load 20 tabs. 19 tabs are inactive.
+        let tabsDatabase = "tabsState20.archive"
+        launchArguments.append(LaunchArguments.LoadTabsStateArchive + tabsDatabase)
+        launchArguments.append(LaunchArguments.LoadDatabasePrefix + tabsDatabase)
+
+        super.setUp()
+
+        // Workaround: Restart app to guarantee the tabs are loaded
+        waitForTabsButton()
+        waitUntilPageLoad()
+        app.terminate()
+        sleep(2)
+        app.launch()
+    }
+
+    // https://mozilla.testrail.io/index.php?/cases/view/2307045
+    func testInactiveTabs() {
+        // Confirm we have tabs loaded
+        let tabsButtonNumber = app.buttons[AccessibilityIdentifiers.Toolbar.tabsButton].staticTexts["20"]
```

Comment:
The test is failing in Bitrise. According to the logs and screenshots, the app is launched with only one tab open :( and so it fails waiting for 20

Reply:
I have opened a new issue with some notes from my investigation. Let me try to run this test on MacStadium.
https://github.com/mozilla-mobile/firefox-ios/issues/22584

Reply:
I have run the test successfully on MacStadium.

Reply:
Let's monitor it so that if it passes consistently we can move it to Smoke. Thanks!